wandb/cli/cli.py

```diff
@@ -1250,11 +1253,58 @@ def human_size(bytes, units=None):
             latest = next(versions)
             print(
                 "{:<15s}{:<15s}{:>15s} {:<20s}".format(
-                    kind.type, latest.updated_at, human_size(latest.size), latest.name
+                    kind.type,
+                    latest.updated_at,
+                    to_human_size(latest.size),
```

Comment:
Not required, but you can use `wandb.util.sizeof_fmt(...)` to do the conversion.  We use it in a bunch of other places.  May be worth througing the reverse logic into wandb.util as well.

Reply:
Good call. Will do.